recipes-devtools/deepstream/deepstream-tao-apps_3.0.bb

```diff
@@ -0,0 +1,38 @@
+DESCRIPTION = "NVIDIA Train, Adapt, and Optimize (TAO) Toolkit sample applications"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE.md;md5=dbef1fb16cd9e5c5249a8e0c5e639fb0"
```

Comment:
Could you also add a `HOMEPAGE =` setting to the recipe?  Even if it just points to the web page for the repo, that's fine.

Reply:
done!